cmd/collection/main.go

```diff
@@ -126,6 +126,7 @@ func main() {
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
 		}).
+		Initialize().
```

Comment:
initializing all node types after all the flags have been defined. 
The access node needs the extra flag to determine how to initialize itself.